configs/server/config.js

```diff
@@ -153,9 +159,9 @@ module.exports =
 	// if you use encrypted private key the set the passphrase
 	tls          :
 	{
-		cert : `${__dirname}/../certs/cert.pem`,
+		cert : `${__dirname}/../../certs/cert.pem`,
```

Comment:
Cert and privkey is not changed so I think you should revert this change.

Reply:
Reverted.